frontend/tests/unit/components/layout/Tooltip.spec.ts

```diff
@@ -100,65 +110,24 @@ describe('Tooltip.vue', () => {
     expect((arrow.element as HTMLElement).style).toContain({ width: '20px', height: '20px' });
   });
 
-  describe('position top', () => {
-    /*
-     * Note that it is currently not possible to test the actual positioning as
-     * in the test environment, the `getBoundingClientRect()` of an HTML element
-     * always returns zero for all properties.
-     */
-
-    it('sets gap as vertical margin to tooltip', async () => {
-      const wrapper = await createWrapper({ gap: '20px', position: 'top', showTooltip: true });
-      const tooltip = wrapper.get('[data-test="tooltip"]');
-
-      expect((tooltip.element as HTMLElement).style).toContain({ margin: '20px 0px' });
-    });
-
-    it('shows the arrow at the center of the bottom border', async () => {
-      const wrapper = await createWrapper({ arrowSize: 20, position: 'top', showTooltip: true });
-      const arrow = wrapper.get('[data-test="arrow"]');
-
-      expect((arrow.element as HTMLElement).style).toContain({
-        bottom: '-10px',
-        right: 'calc(50% - 10px)',
-      });
-    });
-  });
-
-  describe('position left', () => {
-    /*
-     * Note that it is currently not possible to test the actual positioning as
-     * in the test environment, the `getBoundingClientRect()` of an HTML element
-     * always returns zero for all properties.
-     */
-
-    it('sets gap as horizontal margin to tooltip', async () => {
-      const wrapper = await createWrapper({ gap: '20px', position: 'left', showTooltip: true });
-      const tooltip = wrapper.get('[data-test="tooltip"]');
-
-      expect((tooltip.element as HTMLElement).style).toContain({ margin: '0px 20px' });
+  describe('position bottom when not fitting in viewport', () => {
```

Comment:
I think we can rephrase & restructure the describe block here like so: 
- "when not fitting in viewport" 
  - "it positions itself on the bottom"
  - "sets gap ..." 
  - ...
- "when fitting in viewport"
  - "it positions itself on the right"
  - "sets gap ..."
  - ...

When nesting such `describe` blocks i always try to imagine the `describe` block as an environment under which the tests inside of it are made unless the `describe` block is targeting/defining a function/prop/variable name or a logic scope. 
If i imagine it as an environment i always end up in phrases like "when ...." which provides higher value to the tester/user running the tests.
In this case, i also believe the `'position bottom when not fitting in viewport'` is also a testable case and should not be used as a phrase for the `describe` block but instead there should be a test for the `position`.

What do you think?

Reply:
Sounds like a good approach to me. I just adapted the old tests and didn't think to much about the naming. But I like your approach.

Reply:
Only thing is that I think it is not really possible to test the position because `getBoundingClientRect` does always return only 0 for all properties, like Thore's comment suggests. But will think again about it.

Reply:
With the behavior of the `getBoundingClientRect` I couldn't find a way to to test the position properly. With mocking I feel like I just end up testing the mocked value. But added the expected position as a comment in the tests.

Reply:
That's understandable. No need to waste more time on that. :heavy_check_mark: